Update new messages and their translations

3d-gussner opened the pull request:
As with FW3.9.1 some new messages have been introduced here an updated translation version.

3d-gussner committed:
Add missing translations

Commit:
Add missing CZ and IT translations

3d-gussner committed:
Update CZ, FR, IT, ES translations
CZ thanks to @DRracer
FR thanks to Carlin Dcustom
ES tried myself
IT thanks to @wavexx

Only missing is PL

Commit:
Improve wording +change keys

Commit:
Add missing PL translations

3d-gussner committed:
Fix copy paste error in Italian
fix double translations

3d-gussner committed:
Merge branch 'MK3_FW391_languages' of https://github.com/3d-gussner/Prusa-Firmware into MK3_FW391_languages

Commit:
Make PL translation shorter

Commit:
Merge branch 'MK3_FW391_languages' of github.com:3d-gussner/Prusa-Firmware into MK3_FW391_languages

3d-gussner committed:
Fix some length issues and capital letters

3d-gussner committed:
Merge branch 'MK3_FW391_languages' of https://github.com/3d-gussner/Prusa-Firmware into MK3_FW391_languages

3d-gussner committed:
Fixed again translations length issues
updated po files

awenelo commented on lang/lang_en_fr.txt:
```diff
@@ -894,6 +902,10 @@
 
+# c=20 r=4
+"Please check the IR sensor connection, unload filament if present."
+"SVP, verifiez la connexion du capteur IR et si le filament est decharge."
```
I think that this should be `et decharge le filament`("and unload filament", instead of "check if the filament is unloaded")

DRracer replied:
is it shorter than before ;) ?

DRracer replied:
`SVP, verifiez la connexion du capteur IR et decharge le filament.`
this way?

awenelo replied:
Yup!
65 vs 72 characters
```
New
SVP, verifiez la connexion du capteur IR et decharge le filament.
Old
SVP, verifiez la connexion du capteur IR et si le filament est decharge.
```

3d-gussner replied:
Also used something similar in German translation as well as it makes it shorter and it makes sense
@awenelo Thanks again

awenelo replied:
You're welcome!

Commit:
Update FR translation - thanks @awenelo

wavexx committed:
Fix execution of lang-check.py

- Make lang-check.py executable
- Execute directly instead of specifying the python interpreter manually
  ("python" is no longer available on Debian, and would default to
  version 2 prior to that despite being written for python 3)

wavexx committed:
Fix permissions of translation files
